lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1894,27 +1915,30 @@ def build_course(self):
         # pylint: disable=attribute-defined-outside-init
 
         self.course_key = SlashSeparatedCourseKey("Robot", "999", "Test_Course")
-        self.course = CourseFactory.create(org='Robot', number='999', display_name='Test Course')
+
+        with self.store.bulk_operations(self.course_key, emit_signals=False):
```

Comment:
Ack, I must have screwed up something while playing with branches -- this whole thing is in a screwy state. I'll close this PR until I have time to go back and fix it.